packages/react/src/experimental/Navigation/Header/ProductUpdates/index.tsx

```diff
@@ -149,7 +149,7 @@ const ProductUpdates = ({
 const FeaturedDropdownItem = ({
   title,
   href,
-  imageURL,
+  mediaURL,
```

Comment:
I checked our usual naming and we use lowercase in those cases `mediaUrl`. Let's rename it